lib/ohai/plugins/linux/filesystem.rb

```diff
@@ -142,36 +143,48 @@ def generate_mountpoint_view(fs)
       end
     end
 
-    have_lsblk = File.exist?("/bin/lsblk")
+    # We used to try to decide if we wanted to run lsblk or blkid
+    # but they each have a variety of cases were they fail to report
+    # data. For example, there are a variety of cases where lsblk won't
+    # report unmounted filesystems, but blkid will. And vise-versa. Sweet.
+    # So for reliability, we'll run both, if we have them.
+
+    have_lsblk = File.exist?("/bin/lsblk") || File.exist?("/usr/bin/lsblk")
```

Comment:
If we're expanding the search path, should this use one of the various incarnations of `which()`?